code/LeftAndMain.php

```diff
@@ -187,10 +187,7 @@ class LeftAndMain extends Controller implements PermissionProvider
      * @config
      * @var array
      */
-    private static $admin_themes = [
-        'silverstripe/admin:cms-forms',
-        SSViewer::DEFAULT_THEME,
-    ];
+    private static $admin_themes;
```

Comment:
Add a note in the PHPDoc that this is set in `_config/themes.yml` - else it just looks like we don't have a default value here.

Reply:
Probably also assign an empty array by default
```suggestion
    private static $admin_themes = [];
```

Reply:
Done @GuySartorelli